packages/engine-server/src/markdown/remark/dendronPreview.ts

```diff
@@ -30,6 +30,32 @@ export function makeImageUrlFullPath({
   node.url = fpath;
 }
 
+/**
+ * Transforms any wiklinks into a vscode command URI for gotoNote.
+ */
+function commandUriForWikilink({
```

Comment:
nitpick: the name sounds like it returns the command URI, when it actually sets it

Reply:
amended to `modifyWikilinkValueToCommandUri()`